Utility.h

```diff
@@ -3,6 +3,7 @@
 #include <vector>
 #include <memory>
 #include "Item.h"
+#include "Character.h"
```

Comment:
- [ ] FIXME: you can forward declare `Character` since you're not using anything in it in this file.